src/main/java/com/gmail/nossr50/util/Misc.java

```diff
@@ -128,6 +128,20 @@ public static void spawnItemsFromCollection(@Nullable Player player, @NotNull Lo
         }
     }
 
+    /**
+     * Drops the item from the item stack only if it is a sapling (or equivalent)
+     * Needed for TreeFeller
+     */
+    public static void spawnItemIfSapling(@NotNull Player player, @NotNull Location location, @NotNull Collection < ItemStack > drops, @NotNull ItemSpawnReason itemSpawnReason) {
+        String dropString;
+        for (ItemStack drop : drops) {
+            dropString = drop.getType().getKey().getKey();
+            if (dropString.contains("sapling") || dropString.contains("propagule")) {
+                Misc.spawnItem(player, location, drop, itemSpawnReason);
```

Comment:
you're already in Misc so you can drop `Misc.`

Reply:
lol. will do.  I wasn't in Misc at the point I wrote that bit xD

Reply:
done